bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx60.html

```diff
@@ -0,0 +1,124 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+ {% from "macros.html" import google_play_button, send_to_device with context %}
+
+ {% add_lang_files "firefox/whatsnew_59" "firefox/mobile" "firefox/sendto" %}
```

Comment:
No, because people will get to this page based on a list given to RelEng that [takes it into account](https://bugzilla.mozilla.org/show_bug.cgi?id=1457024#c6).

Reply:
Yep, we should be fine on locales as RelEng is handling it.